src/data/image.js

```diff
@@ -15,8 +15,21 @@ export type ImageDetails = {
 };
 
 export const getImageDetails = (images: Images) => (
-  id: string
-): ?ImageDetails => images[id];
+  id: string,
+  dimensions: ?{ width: number, height: number }
+): ?ImageDetails => {
+  if (!dimensions) {
+    return images[id];
+  }
+
+  const imageDetails = images[id];
+  const imageUriWithResizingBehaviourParameters = `${imageDetails.uri}?w=${
```

Comment:
I would not count on the image details being present all the time, i.e. `imageDetails` might be `undefined`.

I think it might be possible in Contentful to select an image in an event, which is still a draft. This means we get the event with the image id. But we don't load the image because we only download published images.